src/components/inspector/encrypted-config.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  
+    <div>
+      <div class="form-group">
+        <b-form-checkbox
+          v-model="settings.encrypted"
+          @change="emitChanges"
+          switch
+        >
+          {{ $t("Encrypted") }}
+        </b-form-checkbox>
+    </div>
+  </div>
+  
+</template>
+
+<script>
+export default {
+  props: {
+    label: String,
+    value: {
+      type: Object,
+      default: () => {
+        return { encrypted: false };
+      }
+    },
+    helper: String,
+  },
+  watch: {
+    value: {
```

Comment:
is it possible to do not use watcher? it cause the reactivity to be hard to track

Reply:
this snippet of code is similar to all another fields attributes used in the srceen builder